### DISCUSSION TITLE
MacOS High Sierra 10.13 support

### DISCUSSION THREAD

**Discussion by Jamskijam** [rustdesk/rustdesk#2955] [General]
Hi! What version supporting MacOS High Sierra 10.13?

**Comment by rustdesk** [comment]
Sadly, no. >=10.14 required.

**Reply by kerta1n** [reply]
Is there a list of supported macOS versions? If not, what minimum version is required for RustDesk to run?

**Reply by Jamskijam** [reply]
Anybody compile support version of 10.13?

**Reply by Koala166** [reply]
I'd like that too! I am using High Sierra and I need RuskDesk.
